server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -27,7 +27,7 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <form id="form-select-characteristic" method="get" novalidate>
+      <form id="form-select-characteristic" method="get" action="/analytics/protected-characteristic" novalidate>
```

Comment:
Needed to include this so when a user searches by a different PC then the anchor tag is cleared